app/src/app/constants/configuration.ts

```diff
@@ -1,14 +1,1935 @@
 import { LngLatLike } from "maplibre-gl";
-import type { MapOptions } from "maplibre-gl";
+import type { MapOptions, StyleSpecification } from "maplibre-gl";
 
-const PUBLIC_MAPTILER_API_KEY = "WlatIY6MghFCwInJhBkl";
-const MAPTILER_API = process.env.NEXT_PUBLIC_MAP_API || PUBLIC_MAPTILER_API_KEY;
-
-export const MAP_TILES = `https://api.maptiler.com/maps/dataviz/style.json?key=${MAPTILER_API}`;
 export const MAP_CENTER: LngLatLike = [-105.358887, 39.113014]; // colorado
 
+const BASEMAP_LAYERS = [
```

Comment:
Should maybe load as JSON?

Reply:
perhaps yes since it's so detailed, if you didn't know `MAP_OPTIONS` was below, i could see someone missing it entirely